packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -227,12 +227,14 @@ export default function createConnector(connectorDesc) {
           metadata,
           resultsFacetValues,
           searchingForFacetValues,
+          isSearchStalled,
         } = store.getState();
         const searchState = {
```

Comment:
We can take advantage of your changes for rename the variable to `searchResults`. The third parameter pass to `getProvidedProps` is `searchResults`. The second one is `searchState`. So it's bit confusing to have `searchResults` called `searchState` 🙂

> https://community.algolia.com/react-instantsearch/guide/Custom_connectors.html#getprovidedpropsprops-searchstate-searchresults-meta-searchforfacetvaluesresults